examples/suit_update/README.hardware.md

```diff
@@ -277,8 +277,7 @@ Devices retrieve the new firmware using the CoAP protocol.
 
 Start `aiocoap-fileserver`:
 
-    $ mkdir -p coaproot
-    $ aiocoap-fileserver coaproot
```

Comment:
This was actually correct, I just executed it inside of `examples/suit_update` :sweat_smile: